Monika After Story/game/zz_submods.rpy

```diff
@@ -101,6 +101,14 @@ init -991 python in mas_submod_utils:
             except:
                 raise SubmodError("Version number '{0}' is invalid.".format(version))
 
+            #Make sure author and name are proper label names
+            import re
```

Comment:
put the import at the top of the store module so its not importing everytime a submod object is created